craftai/client.py

```
@@ -162,6 +186,25 @@ def delete_agent(self, agent_id):
 
     return decoded_resp
 
+  def delete_agents_bulk(self, payload):
+    # payload = [{"id": agent_id}]
+    # Check all ids, raise an error if all ids are invalid
```

So we authorize some id's to fail ? Wouldn't it be better to raise an error if 'ANY' id is invalid ?